t/34-git.t

```diff
@@ -66,12 +63,9 @@ my $case_dir = "file://$git_dir#abcdef";
 
 subtest 'failing clone' => sub {
     %bmwqemu::vars = (CASEDIR => $case_dir);
-    my $path;
     my $out = combined_from {
-        eval { $path = checkout_git_repo_and_branch('CASEDIR', retry_count => 0) };
+        throws_ok { heckout_git_repo_and_branch('CASEDIR', retry_count => 0) } qr{Could not find 'abcdef' in complete history in cloned Git repository "\Q$case_dir\E"}, "Error message when trying to clone wrong git hash"
```

Comment:
```suggestion
        throws_ok { checkout_git_repo_and_branch('CASEDIR', retry_count => 0) } qr{Could not find 'abcdef' in complete history in cloned Git repository "\Q$case_dir\E"}, "Error message when trying to clone wrong git hash"
```

Reply:
awesome. And I thought my t/34-git.t is just completely broken locally :D

 Fixed